src/index.ts

```diff
@@ -866,6 +851,58 @@ class NixInstallerAction extends DetSysAction {
     return;
   }
 
+  async probeDirExistOnHost(directory: string): Promise<boolean> {
+    actionsCore.debug("Probing for {}...");
+
+    const runOutput = await actionsExec.getExecOutput(
+      "docker",
+      ["--log-level=debug", "container", "create", "--init"]
+        .concat([
+          "--mount",
+          `type=bind,src=${directory},dst=${directory},readonly`,
+        ])
+        .concat(["determinate-nix-shim:latest"]),
```

Comment:
```suggestion
      [
        "--log-level=debug",
        "container",
        "create",
        "--init",
        "--mount",
        `type=bind,src=${directory},dst=${directory},readonly`,
        "determinate-nix-shim:latest"
      ],
```

Reply:
Why not just do `--mount=...` in that case?

Reply:
fixed :